examples/ignition/k8s-master.yaml

```diff
@@ -157,7 +157,6 @@ storage:
               - --allow-privileged=true
               - --service-cluster-ip-range={{.k8s_service_ip_range}}
               - --secure-port=443
-              - --advertise-address={{.ipv4_address}}
```

Comment:
Removing this flag should mean the address bound to the default interface is advertised. http://kubernetes.io/docs/admin/kube-apiserver/